enchant.yaml

```diff
@@ -5,7 +5,7 @@ config-opts:
   - --with-myspell-dir=/app/share/myspell
 sources:
   - type: archive
-    url: https://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz
+    url: https://src.fedoraproject.org/lookaside/extras/enchant/enchant-1.6.0.tar.gz/de11011aff801dc61042828041fb59c7/enchant-1.6.0.tar.gz
```

Comment:
you could also use the 1.6.1 tarball from the Github releases (official)

https://github.com/AbiWord/enchant/releases/tag/enchant-1-6-1
https://github.com/AbiWord/enchant/releases/download/enchant-1-6-1/enchant-1.6.1.tar.gz